components/reset_status_modal/index.ts

```diff
@@ -11,17 +11,17 @@ import {getStatusForUserId} from 'mattermost-redux/selectors/entities/users';
 
 import {autoResetStatus} from 'actions/user_actions.jsx';
 
-import ResetStatusModal from './reset_status_modal.jsx';
+import ResetStatusModal from './reset_status_modal';
 
-function mapStateToProps(state) {
+function mapStateToProps(state: any) {
```

Comment:
Please try to avoid 'any'. See other components that are already typescript for example of properly typing 'state' parameter. (the type is called GlobalState)